.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please, not in this patch :)

Reply:
This feels relevant/important as it's the reason I missed the clippy warning above?

Reply:
@cylewitruk The universal way would be adding GitHub workflows to fail the build. E.g. I use VSCode, but I set up `clippy` differently, and your patch conflicts with my settings. Either way, it must be another PR.

Reply:
@fadeevab okay, no problem, I'll remove it :) 
I removed it simply because I didn't see the warning -- I'm rather used to the `settings.json` being present with `clippy` set as the default, so I guess I just assumed that clippy wasn't actually complaining in this case :shrug:

Reply:
Done :)